NurVo_Frontend/src/pages/SignUp.tsx

```diff
@@ -153,54 +155,61 @@ const SignUp = ({ navigation }) => {
   return (
     <>
       <KeyboardAvoidingView
-        behavior={Platform.select({ ios: 'padding', android: 'padding' })}
-        style={styles.container}
-      >
-        <SignUpCell
-          title="아이디"
-          initialText="아이디"
-          isConfirmButton={true}
-          buttonText="중복확인"
-          onText={value => setId(value)}
-          onClickAction={onClick}
-          isButtonDisable={checkId}
-        />
-        <SignUpCell
-          title="비밀번호"
-          initialText="비밀번호"
-          onText={value => setPassword(value)}
-          subText='영어 소문자, 숫자, 특수 기호(!@#$%^&*)을 포함한 8자 이상'
-        />
-        <SignUpCell
-          title="비밀번호 확인"
-          initialText="비밀번호 확인"
-          onText={value => setConfirmPassword(value)}
-        />
-        <SignUpCell
-          title="이름"
-          initialText="이름을 입력하세요"
-          onText={value => setName(value)}
-        />
-        <SignUpCell
-          title="휴대폰 번호"
-          initialText="'-' 구분없이 입력"
-          isConfirmButton={true}
-          buttonText="인증번호 전송"
-          onText={(value) => setPhoneNumber(value)}
-          onClickAction={onClick}
-          isButtonDisable={identify}
-        />
-        <SignUpCell
-          title="닉네임"
-          initialText="닉네임을 입력하세요"
-          onText={value => setNickname(value)}
-        />
+        style={[styles.container]}
+        behavior={Platform.OS === 'ios' ? 'padding' : undefined}
+        keyboardVerticalOffset={Platform.OS === 'ios' ? StatusBarManager.HEIGHT + 44 : undefined}>
+        <ScrollView>
```

Comment:
<ScrollView>
```suggestion
        <View style={styles.container}>
```
로 바꿔주세요!! 회원가입 페이지가 위로 쏠려서요!!
 
![image](https://github.com/CSID-DGU/2023-S-VSA-CoCo-1/assets/127061457/8fe4d096-d5ae-4ce8-bd7c-66ad142dcd7d)

Reply:
사용자 편의 이슈로 스크롤뷰는 필요할 것 같습니다. 대신 네비게이션 바와 셀 사이의 간격을 조정해보도록하겠습니다.

Reply:
![image](https://github.com/CSID-DGU/2023-S-VSA-CoCo-1/assets/127061457/eef3bb1e-898f-439d-a4e9-2b9176b8b2c4)

View로 해도 잘 되긴 하는데, 사용자가 보기에 불편함이 없다면, 간격 조정도 나쁘지 않을 거 같습니다